mediaflow_proxy/utils/hls_prebuffer.py

```
@@ -318,4 +349,142 @@ async def close(self) -> None:
+    def _parse_target_duration(self, playlist_content: str) -> Optional[int]:
+        """
+        Parse EXT-X-TARGETDURATION from a media playlist and return duration in seconds.
+        Returns None if not present or unparsable.
+        """
+        for line in playlist_content.splitlines():
+            line = line.strip()
+            if line.startswith("#EXT-X-TARGETDURATION:"):
+                try:
+                    value = line.split(":", 1)[1].strip()
+                    return int(float(value))
+                except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)